src/support/has.ts

```diff
@@ -7,6 +7,14 @@ export * from 'dojo-has/has';
 
 /* ECMAScript 6 and 7 Features */
 
+/*
+ * Determine whether or not native Symbol exists. If it doesn't, we don't want to use
+ * a few other native implementations like Map, WeakMap, Set.  Consider a polyfill that provides Symbol,
+ * Map, etc in the global namespace. If the polyfill's Symbol is not compatible with our Symbol, neither
+ * will be anything that uses their iterator symbol, like Map, Set, etc.
+ */
+const hasSymbol = typeof global.Symbol !== 'undefined' && typeof Symbol() === 'symbol';
```

Comment:
I would just put this in the `add('es6-symbol');` below... since a lot of them aren't functions though and will evaluate immediately, I would just move it up to here and then replace the uses of `hasSymbol` with `has('es6-symbol');`  I think it is just "cleaner" and leverages the API we have.